scripts/gui/thrive_gui.html

```diff
@@ -527,7 +527,7 @@
 
         <div id="microbeEditorLeftPanel" class="Panel">
 	      <div id="StructurePanelTop" class="CellTabButton">STRUCTURE</div>
-	      <div id="AppearanceButton" class="CellTabButton">APPEARANCE</div>
+	      <div id="MembraneButton" class="CellTabButton">MEMBRANE</div>
```

Comment:
I think the tab shouldn't be renamed, it still has the colour and in concepts the membrane type has been kept under the appearance, though maybe @buckly90  has another idea.

Reply:
Appearance is fine but I feel that it's name conveys that the contents will not effect a cell's viability. Besides, as far as I am aware, the only feature we will have that actually changes a cell's appearance without impacting it's stats is color which would still be right at home in membrane settings. I personally support changing it to membrane.